cmd/iamctl/internal/examplecmd/exampleservercmd/server.go

```diff
@@ -28,13 +27,9 @@ func newServer(spannerClient *spanner.Client) (*iamexample.Authorization, error)
 	if err != nil {
 		return nil, err
 	}
-	roles, err := iamregistry.NewRoles(iamDescriptor.PredefinedRoles)
-	if err != nil {
-		return nil, err
-	}
 	iamServer, err := iamspanner.NewIAMServer(
 		spannerClient,
-		roles,
+		iamDescriptor.PredefinedRoles.Role,
```

Comment:
should this be singular?

Reply:
it's a protobuf descriptor ism where you say a repeated field multiple times as singular:

```
role {
}
role {
}
```

etc

Reply:
follow-up here: https://github.com/einride/iam-go/pull/63